README.md

```diff
@@ -4,11 +4,12 @@
 
 # swagger-php
 
-Generate interactive [OpenAPI](https://www.openapis.org) documentation for your RESTful API using
-[doctrine annotations](https://www.doctrine-project.org/projects/annotations.html) (optional as of version 4.8; if required the `doctrine/annotations` library must be installed in addition to swagger.php).
-or [PHP attributes](https://www.php.net/manual/en/language.attributes.overview.php).
+Generate interactive [OpenAPI](https://www.openapis.org) documentation for your RESTful API using [PHP attributes](https://www.php.net/manual/en/language.attributes.overview.php) (preferred) or 
+[doctrine annotations](https://www.doctrine-project.org/projects/annotations.html) (requires additional `doctrine/annotations` library).
 
-For a full list of supported annotations, please have look at the [`OpenApi\Annotations` namespace](src/Annotations) or the [documentation website](https://zircote.github.io/swagger-php/guide/annotations.html).
+See the [documentation website](https://zircote.github.io/swagger-php/guide/attributes.html) for supported attributes and annotations.
+
+Annotations, and support for PHP versions lower than 8.1 are deprecated and will be removed in a future release of swagger-php.
```

Comment:
Not quite sure about this one, in particular referencing a specific PHP version.
There are already two mentions of 8.1 as the min. required PHP version for attributes, so this doesn't really add anything.

`may be removed` is strong enough here, seeing that annotations are still widely used, IMO. If this changes faster than expected we can still change it.


```suggestion
Annotations are deprecated and may be removed in a future release of swagger-php.
```

Reply:
Thank you, baked in